docs/data/joy/components/modal/modal.md

```diff
@@ -8,113 +8,139 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/
+
+In Joy UI, modals are managed through three related components:
 
 - [`Modal`](#basic-usage): A container that renders its `children` node in front of a backdrop component.
 - [`ModalClose`](#dialog): A button for closing the modal.
 - [`ModalDialog`](#dialog): A component for rendering a modal dialog.
 
 {{"demo": "ModalUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
-### Features
+The Modal component comes equipped with features designed to improve user interaction and accessibility:
 
-- 🥞 Manages modal stacking when more than one is needed.
-- 🪟 Automatically creates a backdrop element to disable interaction with the rest of the app.
-- 🔐 Disables page scrolling while open.
-- ⌨️ Manages focus correctly between the modal and its parent app.
-- ♿️ Adds the appropriate ARIA roles automatically.
+- Manages modal stacking when more than one is needed.
+- Automatically create a backdrop element to disable interactions with the rest of the app.
+- Disables page scrolling when open.
+- Manages focus between modal and its parent app.
+- Adds appropriate ARIA roles automatically.
 
 :::info
-The term "modal" is sometimes used interchangeably with "dialog," but this is incorrect.
+The term "modal" can sometimes be used interchangeably with "dialog," but this is incorrect.
 
-A modal [blocks interaction with the rest of the application](https://en.wikipedia.org/wiki/Modal_window), forcing the user to take action.
-As such, it should be used sparingly—only when the app _requires_ user input before it can continue.
-A dialog, on the other hand, may be _modal_ or _nonmodal (modeless)_.
-:::
+A modal [restricts interaction with the rest of the application](https://en.wikipedia.org/wiki/Modal_window), forcing the user to take action.
+Therefore, it should be used sparingly, reserved for instances where the application _requires_ user input to proceed.
 
-## Component
+Dialog, on the other hand, can either be _modal_ or _nonmodal (modeless)_, offering more flexibility in user interactions.
+:::
 
-After [installation](/joy-ui/getting-started/installation/), you can start building with this component using the following basic elements:
+## Basics
 
 ```jsx
 import Modal from '@mui/joy/Modal';
-
-export default function MyApp() {
-  return <Modal>{children}</Modal>;
-}
 ```
 
-### Basic usage
+The `Modal` component is designed to accept just one React element as its child.
+This could be a Joy UI component, such as `Sheet`, or any other custom element.
 
-The Modal accepts only a single React element as a child.
-That can be either a Joy UI component, e.g. [`Sheet`](/joy-ui/react-sheet/), or any other custom element.
+{{"demo": "BasicModal.js"}}
 
-Use the Modal Close component to render a close button that inherits the modal's `onClose` function.
+## Customization
 
-{{"demo": "BasicModal.js"}}
+### Variants
+
+`ModalDialog` supports Joy UI's [global variants](/joy-ui/main-features/global-variants/): .
```

Comment:
`ModalDialog` => Modal Dialog (this suggestion applies across the doc to all component names)

```suggestion
Modal Dialog supports Joy UI's [global variants](/joy-ui/main-features/global-variants/): .
```